usage-examples/go/atlas-sdk-go/README.md

```diff
@@ -0,0 +1,90 @@
+# Atlas SDK for Go
+
+This project demonstrates how to script specific functionality using the Atlas
+SDK for Go. Code examples are used in the Atlas Architecture Center docs, and
+the project is made available in a user-facing repo.
+
+## Project Structure
```

Comment:
This project structure seems super useful! I also think a version of it would be useful in the public-facing README. Can we add this there, omitting the internal-only README detail and the scripts directory?